hawc/apps/mgmt/filterset.py

```diff
@@ -18,12 +19,17 @@ def filter(self, qs, value):
 
 
 class UserTaskFilterSet(BaseFilterSet):
-    assessment_name = df.CharFilter(
-        lookup_expr="icontains", field_name="study__assessment__name", label="Assessment name"
+    search = df.CharFilter(
+        method="filter_search",
+        label="Assessment/Study",
+        help_text="Filter by assessment or study name",
     )
-    study_name = df.CharFilter(
-        lookup_expr="icontains", field_name="study__short_citation", label="Study name"
```

Comment:
I like this change! Let's change 'search' to a more descriptive name though, like "study_assessment_name"